keras_cv/layers/preprocessing/cut_mix.py

```diff
@@ -0,0 +1,178 @@
+import tensorflow as tf
+import tensorflow.keras as keras
+import tensorflow.keras.layers as layers
+import warnings
+
+
+class CutMix(layers.Layer):
+    """
+    CutMix implements the CutMix data augmentation technique as proposed in https://arxiv.org/abs/1905.04899.
+
+    Args:
+        alpha: alpha parameter for the sample distribution.  Defaults 0.8.
```

Comment:
Not a fan of "alpha: alpha parameter for the sample distribution" because as a naive user reading this I don't know what it means or how I'm suppose to pick a value. The argument name and its description should be more actionable.

E.g. see `rate` from `Dropout` (which is a self-descriptive name)

`rate: Float between 0 and 1. Fraction of the input units to drop.`

Reply:
The description should give some basic hints as to how to set the parameter, e.g. "a higher alpha yields X result, a lower alpha yields X result". Ideally some of that info would be conveyed by the parameter name itself -- is there any other possible name that would be more descriptive than "alpha"? 

From our API design guidelines: *"Avoid overly generic names (x, variable, parameter)."* https://github.com/keras-team/governance/blob/master/keras_api_design_guidelines.md